lib/src/utils.dart

```diff
@@ -20,3 +22,17 @@ int rotl32(int val, int shift) {
   var modShift = shift & 31;
   return ((val << modShift) & mask32) | ((val & mask32) >> (32 - modShift));
 }
+
+/// Random byte generator
```

Comment:
Please see https://dart.dev/guides/language/effective-dart/documentation

Reply:
Thanks. I read it actually and must have overlooked that "@params"  are discouraged.

Reply:
Should be fixed now. Let me know if I'm missing something